src/components/shared/FormInputs.jsx

```diff
@@ -61,33 +61,54 @@ Checkbox.propTypes = {
   name: PropTypes.string,
 };
 
-export const RadioGroup = ({
-  title, options, name, selected, onChange, disabled,
-}) => (
-  <div className={disabled ? 'radio-group disabled' : 'radio-group'}>
-    <div className="field-label">{title}:</div>
-    <ul>
-      {options.map((option) => {
-        const uniqueId = `${title}-${option}-${Math.floor(Math.random() * 0xffff)}`;
-        return (
-          <li key={uniqueId}>
-            <input
-              type="radio"
-              id={uniqueId}
-              value={option}
-              name={name}
-              checked={selected.toString() === option.toString()}
-              onChange={onChange}
-              required
-              disabled={disabled}
-            />
-            <label htmlFor={uniqueId}>{option.toString()}</label>
-            <div className="check" />
-          </li>);
-      })}
-    </ul>
-  </div>
-);
+export class RadioGroup extends Component {
+  constructor(props) {
+    super(props);
+    this.onChange = this.onChange.bind(this);
```

Comment:
Are we using `react-autobind` in this project? For consistency, we might want to use it here even though it doesn't save any lines of code...